translations/LICENSE-nb.md

```diff
@@ -0,0 +1,20 @@
+# «IKKE VÆR EN DRITTSEKK»-LISENSEN
+> Versjon 1.1, Desember 2016
+
+> Opphavsrett © [år] [fullt navn]
+
+Alle har tillatelse til å kopiere og distribuere identiske eller modifiserte kopier av dette lisensdokumentet.
+
+> «IKKE VÆR EN DRITTSEKK»-LISENSENS REGLER OG VILKÅR FOR KOPIERING, DISTRISJON, OG MODIFISERING
```

Comment:
DISTRIBUSJON

Reply:
Ah, I must've forgot back then to iron out that spelling error somehow.